morphic.js

```diff
@@ -7818,10 +7820,47 @@ MenuMorph.prototype.select = function (aMenuItem) {
     this.selection = aMenuItem;
 };
 
+MenuMorph.prototype.mouseLeave = function () {
+    if (this.parent instanceof MenuMorph) {
+        // this is a submenu so we vanish when lose focus
```

Comment:
:)

Very annoying that when you type a colon, github fills half your screen with a menu of suggestions, none of which is the one you 95% of the time mean!  :(